packages/core/contracts/common/implementation/Lockable.sol

```diff
@@ -10,21 +10,18 @@ contract Lockable {
     bool private _notEntered;
 
     constructor() {
-        // Storing an initial non-zero value makes deployment a bit more
```

Comment:
this was not related to the audit but the comment line length did not follow our standard.